volumes/volume_contents.txt

```diff
@@ -0,0 +1 @@
+Hello from a mounted volume!
```

Comment:
I'm not crazy about adding volumes as a top level item given that it's not a primary component of Cloud Native Buildpacks. Could this be something that the tutorial guides the user in creating? For example,

> Create the following content that we will mount onto the build process:
> 
> ```shell
> mkdir -p /tmp/my-volume
> echo "Hello from a mounted volume!" > /tmp/my-volume/volume-file.txt
> ```